composer.json

```diff
@@ -11,15 +11,24 @@
     }
   ],
   "require": {
-    "php": ">=5.3.0",
-    "phpunit/phpunit": "3.7.*",
-    "swiftmailer/swiftmailer": ">=v5.2.0"
+    "php": ">=5.3.0"
+  },
+  "require-dev": {
+    "phpunit/phpunit": "^3.7 || ^4.0 || ^5.0",
```

Comment:
Allow for more modern phpunit to be used when available via PHP version constraints.